src/app/vm/vm-actions/vm-actions-component/vm-access.component.html

```diff
@@ -1,5 +1,71 @@
-<cs-postdeployment-dialog
-  [vm]="vm"
-  [dialogRef]="dialogRef"
-  [title]="title">
-</cs-postdeployment-dialog>
+<h3 mat-dialog-title>{{ 'VM_PAGE.COMMANDS.VM_ACCESS' | translate }}</h3>
+<mat-tab-group>
+  <mat-tab mat-tab-class="my-class" [label]="'VM_POST_ACTION.VNC_CONSOLE' | translate">
+    <div class="parameter-wrapper">
+      <p><b>{{ 'VM_POST_ACTION.LOGIN' | translate }}:</b> {{ vncAccessService.getLogin(vm) }}</p>
+      <p><b>{{ 'VM_POST_ACTION.VM_PASSWORD' | translate }}:</b> {{ getVncPassword(vm) }}</p>
+    </div>
+
+    <p class="action-link">
+      <a (click)="openConsoleVm(vm)">{{ 'VM_POST_ACTION.OPEN_VNC_CONSOLE' | translate }}</a>
+    </p>
+  </mat-tab>
+
+  <mat-tab [disabled]="!sshAccessService.isSshAuthMode(vm)" [label]="'VM_POST_ACTION.SHELL_CONSOLE' | translate">
+    <div class="parameter-wrapper">
+      <p>
+        <b>{{ 'VM_POST_ACTION.CONNECTION_STRING_4' | translate }}:</b>
+        {{ sshAccessService.getIpv4ConnectionString(vm) }}
+        <cs-clipboard-button
+          cs-input-group-button
+          [value]="sshAccessService.getIpv4ConnectionString(vm)"
+          tooltipPosition="below"
+        ></cs-clipboard-button>
+      </p>
+      <p *ngIf="vm.nic[0]?.ip6address">
+        <b>{{ 'VM_POST_ACTION.CONNECTION_STRING_6' | translate }}:</b>
+        {{ sshAccessService.getIpv6ConnectionString(vm) }}
+        <cs-clipboard-button
+          cs-input-group-button
+          [value]="sshAccessService.getIpv6ConnectionString(vm)"
+          tooltipPosition="below"
+        ></cs-clipboard-button>
+      </p>
+      <p *ngIf="vm.nic[0]?.ipaddress"><b>{{ 'VM_POST_ACTION.IPv4' | translate }}:</b> {{ vm.nic[0].ipaddress }}</p>
```

Comment:
Can an IPv4 address not exist? If yes, then why you not to check this in the connection string?